package.json

```diff
@@ -2,9 +2,13 @@
   "name": "@stackone/ai",
   "version": "0.2.0",
   "description": "Tools for agents to perform actions on your SaaS",
-  "module": "dist/index.js",
-  "main": "dist/index.js",
-  "types": "dist/index.d.ts",
+  "module": "./dist/index.js",
+  "main": "./dist/index.js",
+  "types": "./dist/index.d.ts",
+  "exports": {
+    ".": "./dist/index.js",
+    "./package.json": "./package.json"
+  },
```

Comment:
added exports 

IMO, instead of exporting all modules from index.ts, it is better if we export modules like `@stackone/ai/type` and `@stackone/ai/toolset` or whatever